src/api/selection.js

```diff
@@ -101,35 +101,36 @@ define(function () {
         nodeHelpers.getAncestor(node, scribe.el, nodeFilter);
     };
 
-    Selection.prototype.placeMarkers = function () {
+    Selection.prototype.isInScribe = function () {
```

Comment:
The diff doesn't really show this nicely but `placeMarkers` still exists but it has functionality to check whether the selection was in Scribe that I broke out into a method that could be used in `create-link`.